src/Controls/src/Core/Handlers/Items/iOS/MauiCollectionView.cs

```diff
@@ -53,24 +51,13 @@ public override void MovedToWindow()
 		base.MovedToWindow();
 		_movedToWindow?.Invoke(this, EventArgs.Empty);
```

Comment:
Can you make the _movedToWindow implementation on MauiCollectionView use a weak event manager?